ColorPatterns/Settings/SettingsBundleHelper.swift

```diff
@@ -8,16 +8,16 @@
 
 import Foundation
 
-class SettingsBundleHelper {
+final class SettingsBundleHelper {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/223a37e3a6bb808c9323297228c70c01940ebad2/ColorPatterns/Settings/SettingsBundleHelper.swift#L11"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>